ios/FluentUI/Avatar/Persona.swift

```diff
@@ -42,6 +42,11 @@ public protocol Persona {
 
     /// The subtitle value for the persona.
     var subtitle: String { get }
+
+    /// Fetches the image asynchronously. This is called when setting up persona row for presentation. Image caching should be handled in the implementation for optimal efficiency.
+    /// If the ``image`` is provided, it will be overrided by the fetched image when completion block is called, acting like a custom placeholder.
+    /// - Parameter completion: The completion block that returns the image
+    @objc optional func fetchImage(completion: @escaping (UIImage?) -> Void)
```

Comment:
Thanks for updating the naming here.
I left another message about versioning (we needed to release 0.4.4 and this change will be released in version 0.4.5).
Do you think you can include the demo controller changes in this PR too?

Reply:
Updated with the demo change. 
![Simulator Screen Recording - iPhone 13 Pro - 2022-02-12 at 12 02 59](https://user-images.githubusercontent.com/3644377/153720917-2e2d5246-6db6-4045-8c04-e97e29461aa0.gif)